src/redux/actions.js

```diff
@@ -1,14 +1,20 @@
-export const incrementCount = (date) => ({
+export const incrementCount = (date, id) => ({
```

Comment:
I don't know where I got this from, but when I use ordered params, I have a tendency to order my method params in the order they would be used:

- Values I need in order to find the right object
- Values I need in order to perform the right action on the object
- Values I might not need or use

This is building an action so they all get used at the same time, but they essentially call the reducer so my head still turns to this. So I'd but `id` here first... Or using named params avoids all this too...

Reply:
I've updated the actions to use named parameters, I think that is the best approach as I imagine we will need to pass fuller payloads as the app functionality grows. I shall keep this in mind though if using positional args in future :)